django_ontruck/denormalisation.py

```diff
@@ -0,0 +1,347 @@
+from importlib import import_module
+from re import split
+from uuid import uuid4
+
+from django.db.models import Model, OneToOneField, CASCADE, prefetch_related_objects
+from django.db.models.signals import post_save
+from more_itertools import flatten
+
+
+def capitalize(string):
```

Comment:
You can replace `capitalize` and `camel_case` with this one liner:
```python
''.join("foo bar".title().split())
```

Reply:
This behaves slightly differently in that it maps `denormalised_fields` to `Denormalised_Fields`.